include/swift/AST/Mangle.h

```diff
@@ -154,6 +154,8 @@ class Mangler {
   void mangleProtocolList(ArrayRef<Type> protocols);
   void mangleIdentifier(Identifier ident,
                         OperatorFixity fixity = OperatorFixity::NotOperator);
+  void mangleIdentifier(DeclBaseName ident,
```

Comment:
What do we want to do about things like this? Is continuing to say `mangleIdentifier` the right way forward? I think it's fine for this patch (minimizes an already large changeset), but what do you think for follow-up?